dedibox-network/rpn/reference-content/private-networks-overview.mdx

```diff
@@ -72,7 +72,7 @@ Take a look at our [documentation](/compute/instances/how-to/use-private-network
 
 ### Interconnecting Scaleway Elements and Scaleway Dedibox with Virtual Private Networks
 
-To interconnect your Scaleway Dedibox dedicated servers and your Scaleway Elements instances, you can choose to configure a **VPN**: a **V**irtual **P**rivate **N**etwork. Here, the public Internet interface is used to create an anonymous and encrypted private network. In essence, a VPN creates an encrypted tunnel between you and the internet, routing all your internet traffic through this tunnel and masking your IP address and data as it travels. Since your internet activity is hidden by the VPN, you have enhanced privacy and anonymity. With VPNs, you can create hybrid infrastructures across all of Scaleway's product ranges, even if instances or servers are not in the same availability (geographic) zone.
+To interconnect your Scaleway Dedibox dedicated servers and your Scaleway Elements instances, you can choose to configure a **VPN**: a **V**irtual **P**rivate **N**etwork. Here, the public Internet interface is used to create an anonymous and encrypted Private Network. In essence, a VPN creates an encrypted tunnel between you and the internet, routing all your internet traffic through this tunnel and masking your IP address and data as it travels. Since your internet activity is hidden by the VPN, you have enhanced privacy and anonymity. With VPNs, you can create hybrid infrastructures across all of Scaleway's product ranges, even if instances or servers are not in the same availability (geographic) zone.
```

Comment:
I think here it isn't a product name (only for Elements)

Reply:
```suggestion
To interconnect your Scaleway Dedibox dedicated servers and your Scaleway Elements instances, you can choose to configure a **VPN**: a **V**irtual **P**rivate **N**etwork. Here, the public Internet interface is used to create an anonymous and encrypted private network. In essence, a VPN creates an encrypted tunnel between you and the internet, routing all your internet traffic through this tunnel and masking your IP address and data as it travels. Since your internet activity is hidden by the VPN, you have enhanced privacy and anonymity. With VPNs, you can create hybrid infrastructures across all of Scaleway's product ranges, even if instances or servers are not in the same availability (geographic) zone.
```